PlayCover/Views/AppSettingsView.swift

```diff
@@ -460,6 +463,13 @@ struct MiscView: View {
                         }
                     }
                     Spacer()
+                    Button((app.hasAlias() == true) ? "settings.removeAlias" : "settings.createAlias") {
+                        if app.hasAlias() == false {
+                            shell.createAlias(app)
```

Comment:
I think updating `hasAlias: Bool?` here to true and to false in else statement will fix the issue @ZhichGaming mentioned.

Reply:
And also using it in the button instead of `(app.hasAlias() == true)`. Also it doesn't need an `== true` since it returns a bool.